client/add/add.module.js

```diff
@@ -20,4 +20,17 @@ ng.module('porybox.add', ['porybox.box', 'porybox.pokemon', 'ngMessages', 'ngFil
       '$scope', '$location', 'io', '$mdDialog', '$mdMedia', '$mdToast', 'errorHandler', controller
     ],
     controllerAs: 'add'
+  })
+  .component('addPokemon',
+  {
+    bindings: {
+      boxes: '=',
+      prefs: '=',
+      selected: '='
+    },
+    template: '<md-button class="md-raised md-primary" ng-click="add.pokemon($event)">Add Pokémon</md-button>',
```

Comment:
I'm assuming the reason is that it needs to call through to the add pokemon call on the add controller, which the box list doesn't have access to.

We could either do it this way, or we could pass the function down, which seems a little messy. Honestly, I'm happy with how @Raia has done it.